controller/src/beerocks/bml/rdkb/internal/bml_rdkb_internal.cpp

```diff
@@ -132,7 +132,9 @@ int bml_rdkb_internal::steering_client_set(uint32_t steeringGroupIndex, const BM
         request->config().snrHighXing      = config->snrHighXing;
         request->config().snrLowXing       = config->snrLowXing;
         request->config().authRejectReason = config->authRejectReason;
-        request->remove()                  = 0;
+        if (request->config().snrProbeHWM > 0) {
+            request->remove() = 0;
+        }
```

Comment:
Why specifically snrProbeHWM? Why not other parameters?
I don't understand why it is needed, the API says that if assigning nullptr to the argument `BML_STEERING_CLIENT_CONFIG *config` it means a request for removal.

Reply:
This is how the API is used, softblock is removed when *config is NULL OR if snrProbeHWM=0.
This is NOT a WA, we need to changes the API description as well.

Reply:
> This is how the API is used, softblock is removed when *config is NULL OR if snrProbeHWM=0.
> This is NOT a WA, we need to changes the API description as well.

Please provide new API description and I will add it as part of this pr

Reply:
"   * Call this function to add/modify/remove per-client configuration config of client_mac.
     * To remove a client configuration call with NULL as config or snrProbeHWM=0."